src/App.tsx

```diff
@@ -47,7 +47,7 @@ function LeftSidePanelWrapper() {
   const [primaryPane, setPrimaryPane] = useState<PrimarySideBarPane>('Explorer');
   const openReference = useSetAtom(openReferenceAtom);
 
-  function handleSideBarClick(selectedPane: PrimarySideBarPane) {
```

Comment:
function statements inside other functions are frowned upon because they're hoisted, just like variables declared with `var`.

Reply:
What is wrong with that? Is there any other advantages of using function expression? (I know that is easier to define the function).